server/controllers/project.controller.js

```diff
@@ -63,12 +65,29 @@ export function getProject(req, res) {
     });
 }
 
-export function deleteProject(req, res) {
-  Project.remove({ _id: req.params.project_id }, (err) => {
-    if (err) {
-      return res.status(404).send({ message: 'Project with that id does not exist' });
+function deleteFilesFromS3(files, callback) {
+  each(files, (file, callback) => {
+    if (!(file.url === undefined)) {
+      deleteObjectFromS3(file.url, () => {});
```

Comment:
The callback setup here is not quite right! This should be
```javascript
deleteObjectFromS3(file.url, fileCallback);
```
And then line 73 is extraneous and should be removed.